src/get-started/flutter-for/web-devs.md

```diff
@@ -863,10 +863,10 @@ with some formatting characteristics.
 To display text that uses multiple styles
 (in this example, a single word with emphasis),
 use a [`RichText`][]widget instead.
-Its `text` property can specify one or more [`TextSpan`][] widgets
+Its `text` property can specify one or more [`TextSpan`][]s
```

Comment:
From [style guide](https://developers.google.com/style/code-in-text?hl=en#grammatical-treatment-of-code-elements):

> Don't inflect the name of a code element, such as to make it plural or possessive. Instead, include a noun after the name of the code element, and inflect that noun.

```suggestion
Its `text` property can specify one or more [`TextSpan`][] objects
```

Reply:
Thanks for the reference!